Gopkg.toml

```diff
@@ -0,0 +1,19 @@
+[[constraint]]
+  branch = "master"
+  name = "github.com/dustin/go-humanize"
+
+[[constraint]]
+  name = "github.com/hashicorp/terraform"
+  version = "0.11.7"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/pyToshka/go-virtualbox"
```

Comment:
Could you update this dependency to the root repo? The `pyToshka` version is a fork. I have pulled in all the changes from this fork in the root repo:

root repo = `github.com/terra-farm/go-virtualbox`

Reply:
It was generated by `dep init` based on the code in the repo at the time, specifically:

https://github.com/terra-farm/terraform-provider-virtualbox/blob/master/provider/resource_vm.go#L17

If someone wants to file a PR to change the package (or I can do it), I will update the PR.

Reply:
I will leave it for now. Let's get the build working so that this can be merged.